fabric-item-api-v1/build.gradle

```diff
@@ -2,5 +2,8 @@ archivesBaseName = "fabric-item-api-v1"
 version = getSubprojectVersion(project, "1.2.1")
 
 moduleDependencies(project, [
-		'fabric-api-base'
+		'fabric-api-base',
+		'fabric-content-registries-v0',
+		'fabric-lifecycle-events-v1',
+		'fabric-resource-loader-v0'
```

Comment:
Why does this need resource loader and lifecycle events?

Reply:
Content registries depends on it.

Reply:
Yeah they are not applied recursively

Reply:
I think they are recursive for compilation but not for runtime. So if you don't declare, it crashes your test mod (that's how it was when I last tested)